### DISCUSSION TITLE
Inputting specific surface area as a parameter

### DISCUSSION THREAD

**Discussion by isaacbasil** [pybamm-team/PyBaMM#3927] [Q&A] [answered]
Hi everyone, 

I am modelling a half cell using a DFN model. I want to use the standard approach of solving the spherical diffusion equation for concentration in the particles. However, I don't want the specific surface area to be calculated from the sphere radius (3*eps/radius) - instead I want to input my own value (which could be a measurement from a BET experiment for example).  
It seems that I have to use the option “particle shape” : "no particles". But this seems a bit confusing to me, because I don't want "no particles", as I am still solving the diffusion equation in a spherical particle. Anyway, I have tried using this option, and then adding	'Positive electrode surface area to volume ratio [m-1]' to my parameters file, but this returns an error. Am I using the right approach here? 

Thanks in advance, 

Isaac

**Comment by valentinsulzer** [comment, marked as answer]
In practice the surface area only appears via the exchange-current density, and changing `a` messes up conservation of lithium. So you should change your exchange-current density function instead (scale by your calculated area divided by 3eps/R)

**Reply by valentinsulzer** [reply]
For what I mean by "conservation of lithium" see appendix C of https://iopscience.iop.org/article/10.1149/1945-7111/ac3e48/pdf

**Reply by isaacbasil** [reply]
Thanks, ok I will modify my i0 function! I had a read of the appendix, but I'm not clear what you mean about it messing up the mass balance. Since changing `a` would have the same effect as changing `i0`, and the Li conservation is still respected when i0 is changed, why would changing `a` directly violate the mass balance?

**Reply by valentinsulzer** [reply]
The way the equations are formulated in pybamm, mass balance is only respected if a = eps * V / A (eqn 33 of the appendix). Changing i0 or a has the same effect on the overpotential in Butler-Volmer, but different effect on the mass balance

**Reply by isaacbasil** [reply]
ah ok I see, thanks! (sorry for late reply).
